tsconfig.json

```diff
@@ -7,7 +7,8 @@
     "typeRoots": ["./node_modules/@types", "./types"],
     "declaration": true,
     "esModuleInterop": true,
-    "outDir": "dist"
+    "outDir": "dist",
+    "resolveJsonModule": true
```

Comment:
You can get rid of this as there are no usages of `json` file

Reply:
Nice catch, solved.